src/Compilers/CSharp/Test/Emit/CodeGen/CodeGenAsyncTests.cs

```diff
@@ -13,11 +13,19 @@
 using Roslyn.Utilities;
 using Xunit;
 using Basic.Reference.Assemblies;
+using System.Runtime.CompilerServices;
+using System.Reflection;
```

Comment:
nit: sort.

Reply:
Will do this in a followup unless Rikki wants further changes in this PR